### ISSUE TITLE
Change focus indicator color

### ISSUE THREAD

**Issue by ggiraldez** [instedd/planwise#201]
The focus indicator color for all form controls is the default color (blue). Change it to green (from the site's palette). This should be just setting the `outline-color` CSS property, but double check that it works for all controls (eg. the combo box from re-com might use a different setting).

**Comment by mmuller** [comment]
Ok in version 0.5.1-130-gab8d3ae
